tests/requirements.txt

```diff
@@ -0,0 +1,7 @@
+# NOTE: these requirements.txt contains packages needed for development env
+
+# install ripple_api as a symlink to sources
```

Comment:
When you work on sources of a package which you may require to be "installed" in the system (site-packages for this case), you may want to be able to work with sources in the same time.

This line asks pip to install `ripple_api` packages in `developer` mode, meaning, `ripple_api` is symlinked from `site-packages` to the source, so that any changes in code are reflected in installed module.